FESS-GUI/mainwindow.h

```diff
@@ -40,6 +41,7 @@ class MainWindow : public QMainWindow
     ~MainWindow();
 
 private:
+    CommonDeviceInterface *flywheelCommunicationDevice;
```

Comment:
I know we discussed this in the meeting but I will answer this here for documentation purposes.
I feel it would be better in the main window as it can then be shared between classes and modified if needed. For examples, changing the serial device or changing to the demo device.